pyspark-notebook/Dockerfile

```diff
@@ -46,19 +46,21 @@ USER jovyan
 
 # Install Python 3 packages
 RUN conda install --quiet --yes \
-    'ipywidgets=4.1*' \
+    'ipywidgets=5.1*' \
     'pandas=0.17*' \
     'numexpr=2.5*' \
     'matplotlib=1.5*' \
     'scipy=0.17*' \
     'seaborn=0.7*' \
     'scikit-learn=0.17*' \
     && conda clean -tipsy
-
+# Activate ipywidgets extension in the environment that runs the notebook server
+RUN jupyter nbextension enable --py widgetsnbextension --sys-prefix
+    
 # Install Python 2 packages and kernel spec
 RUN conda create --quiet --yes -p $CONDA_DIR/envs/python2 python=2.7 \
-    'ipython=4.1*' \
-    'ipywidgets=4.1*' \
+    'ipython=4.2*' \
+    'ipywidgets=5.1*' \
```

Comment:
> And, @jakirkham I'm not up to speed on how these get packaged for conda. Is it now an open community effort with forge or is someone doing official builds of major projects like notebook, ipywidgets, etc.

Yeah, there is a lot of discussion on this point. I think the most crucial piece (as we don't all have time to read everything) is this [comment]( https://github.com/conda-forge/conda-forge.github.io/issues/22#issuecomment-197117472 ). There will continue to be a `defaults` channel that is maintained by Continuum. However, I think in practice for projects like Jupyter that release quickly and often, `defaults` won't be the complete solution.

The conda-forge effort is the community driven solution for making and maintaining conda packages. It has been growing quite rapidly. It contains a large portion of what is in [conda-recipes]( https://github.com/conda/conda-recipes ). Also, it is incorporating things where there was no open source recipe previously. I expect we soon will eclipse the `defaults` channel.

The community has also grown quite substantially. Anyone can join and participate as they wish. We have an active working relationship with Continuum and plan to continue that. There are Continuum employees that help with this maintenance burden as well. We have been trying to have regular meetings every two weeks loosely modeled after Jupyter meetings. Details of past and future meetings are in our [hackpad]( https://conda-forge.hackpad.com/conda-forge-meetings-2YkV96cvxPG ).

If there are any more questions, please don't hesitate to ask.